examples/iota_lens/analysis_iotalens.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 ImpactX contributors
+# Authors: Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+
+import glob
+
+import numpy as np
+import pandas as pd
+from scipy.stats import moment
+
+
+def get_moments(beam):
```

Comment:
Do we want to do this and/or (additionally) compare each particle by id (before/after)?